app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] One of the project requirements highlights this 👇🏼:
  
  > - [Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and your styling.
  
  This login view is still not aligning with the wireframe provide for it. The wireframe for this view looks like this:
    
  ![image](https://user-images.githubusercontent.com/45002654/170320186-6bf656dd-d19c-492f-8a5b-84f9f1053be1.png)
  
  But it currently looks like this:
  
  ![image](https://user-images.githubusercontent.com/45002654/170320582-068cbbc6-3ba4-415f-a3d3-8df082fb4f47.png)
  
  Styling is not a core requirement at this stage, but we are at least required to give our views a bare minimum design that's on the wireframes.
  
  Please work on this.